### PR TITLE
Add a useful example for understanding.

### PR DESCRIPTION
**Pull request by Alwaysherewithme**
### Description

Add a more useful example for understanding `:last-of-type` pseudo-classes selector.

### Motivation

The example shown already is good but not suitable for complicated situations. I came across a problem and applied a solution in my example.

### CODE REVIEW

File: files/en-us/web/css/_colon_last-of-type/index.md [hunk 1]
```diff
@@ -79,6 +79,40 @@ article :last-of-type {
 
 {{EmbedLiveSample('Nested_elements', 500)}}
 
+### Multiple selectors elements
+
+This is a bit complicated example of nested elements with type selectors and class selectors.
```
Comment (estelle) [inline files/en-us/web/css/_colon_last-of-type/index.md:84]:
```suggestion
This HTML example contains nested elements of different types. The CSS contains both type selectors and class selectors.
```

File: files/en-us/web/css/_colon_last-of-type/index.md [hunk 2]
```diff
@@ -79,6 +79,40 @@ article :last-of-type {
...
+
+.container .item:last-of-type {
+  color: red;
+}
+```
+
+#### Result
+
+{{EmbedLiveSample('Multiple_selectors_elements', 500)}}
+
```
Comment (estelle) [inline files/en-us/web/css/_colon_last-of-type/index.md:115]:
```suggestion

The last `<div>` and the last `<p>` are both red and bold as the `.item:last-of-type` selects the last of every type if that last element also has the `item` class.
```

File: files/en-us/web/css/_colon_last-of-type/index.md [hunk 3]
```diff
@@ -79,6 +79,40 @@ article :last-of-type {
...
+#### CSS
+
+```css
+p:last-of-type {
+  background: skyblue;
+}
+
+.container .item:last-of-type {
+  color: red;
```
Comment (estelle) [inline files/en-us/web/css/_colon_last-of-type/index.md:108]:
```suggestion
  color: red;
  font-weight: bold;
```

**Review by estelle — COMMENTED** [review]
the example is a good idea. I made some suggestions.

File: files/en-us/web/css/_colon_last-of-type/index.md (hunk 2 above)
Comment (Alwaysherewithme) [reply files/en-us/web/css/_colon_last-of-type/index.md:115]: Great! Thanks @estelle for these helpful details.

**Review by estelle — APPROVED** [review]
Thank you!

**Pull request merged**